### PR TITLE
Release R148

### PR DESCRIPTION
**Pull request by matus-tomlein**
This release add schemas for:

* updated schemas for the Mandrill adapter (reviewed in PR #1342)
* new major version of the browser context schema (reviewed in PR #1343)
* new schema for GA4 cookies (reviewed in #1325)
* new schema for AMP tracker consent (reviewed in #1329)

## Changelog

Add com.google.ga4/cookies/jsonschema/1-0-0 (#1324)
Add schemas/dev.amp.snowplow/amp_consent/jsonschema/1-0-0 (#1328)
Add com.snowplowanalytics.snowplow/browser_context/jsonschema/2-0-0 (#1330)
Add com.mandrill/recipient_unsubscribed/jsonschema/1-0-2 (#1341)
Add com.mandrill/message_soft_bounced/jsonschema/1-0-2 (#1340)
Add com.mandrill/message_sent/jsonschema/1-0-1 (#1339)
Add com.mandrill/message_rejected/jsonschema/1-0-1 (#1338)
Add com.mandrill/message_opened/jsonschema/1-0-2 (#1337)
Add com.mandrill/message_marked_as_spam/jsonschema/1-0-2 (#1336)
Add com.mandrill/message_delivered/jsonschema/1-0-0 (#1335)
Add com.mandrill/message_delayed/jsonschema/1-0-2 (#1334)
Add com.mandrill/message_clicked/jsonschema/1-0-2 (#1333)
Add com.mandrill/message_bounced/jsonschema/1-0-2 (#1332)

### CODE REVIEW

File: schemas/com.mandrill/message_clicked/jsonschema/1-0-2 [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+		"url": {
+			"type": "string",
+			"description": "The url clicked for the event"
+		},
+		"user_agent_parsed": {
+			"type": ["object", "null"],
+			"description": "A parsed version of the user agent detected for the event. Value will be null if the user agent can't be determined.",
+			"properties": {
+				"mobile": {
+					"type": ["boolean", "null"],
```
Comment (adatzer) [inline schemas/com.mandrill/message_clicked/jsonschema/1-0-2:224]: Just flagging to get confirmation on this: 
The `user_agent_parsed` properties have all been optional. In 1-0-1 version their type was just `boolean` or `string`. In version 1-0-2, these properties are still optional, but their type is extended to include `"null"`. 
Is this ok for a patch change?

File: schemas/com.google.ga4/cookies/jsonschema/1-0-0 [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        "session_cookies": {
+            "description": "Google Analytics 4 session cookie entries.",
+            "type": [ "array", "null"],
+            "items": {
+                "type": "object",
+                "description": "GA4 session cookie entry.",
+                "properties": {
+                    "measurement_id": {
+                        "description": "Identifier for the Google Analytics 4 container to which the session cookie belongs to",
+                        "type": "string",
```
Comment (adatzer) [inline schemas/com.google.ga4/cookies/jsonschema/1-0-0:31]: nit - to be consistent with the other optional props to go for `["string", "null"]` ?

**Review by adatzer — APPROVED** [review]
LGTM!

**Review by istreeter — APPROVED** [review]
All looks good to me!

This PR adds an optional field to a couple of existing schemas.  We have talked about this before in previous Iglu Central PRs -- it can potentially cause loading issues in the Redshift loader, because of a race condition that we still haven't fixed.  In previous Iglu Central releases we decided to go ahead and release the schema anyway, so I think we can do the same thing here.

I guess these mandrill schemas are rarely used -- not many redshift pipelines will have tables for these schemas.  So the chance of hitting an error is very very slim.

Even so, we should release it carefully, and watch out for loading errors over the following hours.  Any loading error is easily fixed, it's just an annoyance.

**Comment by matus-tomlein** [comment]
Added a schema from @oguzhanunlu that was reviewed in #1350. @istreeter has already approved this and is aware of it being added to the release.

File: schemas/com.snowplowanalytics.monitoring.batch/load_succeeded/jsonschema/3-0-1 [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+      "format": "date-time"
+    },
+    "loadingCompleted": {
+      "description": "Time when loader finished loading (should match the ingestion timestamp in manifest)",
+      "type": "string",
+      "format": "date-time"
+    },
+    "recoveryTableNames": {
+      "description": "List of recovery table names loaded in this batch",
+      "type": "array",
```
Comment (jbeemster) [inline schemas/com.snowplowanalytics.monitoring.batch/load_succeeded/jsonschema/3-0-1:180]: As this is a new-field doesn't this need to be `nullable` or does `array` type implictly imply optionality?

**Pull request closed without merging**